lib/foreman_tasks/dynflow.rb

```diff
@@ -91,6 +91,9 @@ def web_console
           end
         end
 
+        set(:custom_navigation) do
+          { "Back to Foreman" => "/#{ForemanTasks::TasksController.controller_path}" }
```

Comment:
If we use don't use the Foreman name, we will avoid branding problems later, "Back to tasks" sounds equally to me, while avoiding the issues